tensorflow-base-cpu-linwin/meta.yaml

```diff
@@ -123,7 +119,6 @@ requirements:
     - numpy >=1.20  # [not aarch64]
     - numpy >=1.19  # [aarch64]
     - opt_einsum 3.3.0.*
-    - protobuf >=3.9.2
     - six >=1.15.0
     - termcolor >=1.1.0
     - typing_extensions >=3.7.4
```

Comment:
typing_extensions >= 3.6.6, see https://github.com/tensorflow/tensorflow/blob/2ea19cbb575d076b4f521d3603211c8316ad5f8f/tensorflow/tools/pip_package/setup.py#L95

Reply:
I think we do not need `wheel` in `run`, see https://github.com/tensorflow/tensorflow/blob/2ea19cbb575d076b4f521d3603211c8316ad5f8f/tensorflow/tools/pip_package/setup.py#L73

Reply:
`grpcio` pinnings https://github.com/tensorflow/tensorflow/blob/2ea19cbb575d076b4f521d3603211c8316ad5f8f/tensorflow/tools/pip_package/setup.py#L127